extensions/amp-timeago/validator-amp-timeago.protoascii

```diff
@@ -42,9 +42,11 @@ tags: {  # <amp-timeago>
   attrs: { name: "locale" }
   attr_lists: "extended-amp-global"
   cdata: {
-    # Regex: \s*[\w.,:/+-]+(\s*[\w.,:/+-]+)*\s*
+    # Regex: \s*[\w\p{L}\p{N}_.,:/+-]+(\s*[\w\p{L}\p{N}_.,:/+-]+)*\s*
     # CDATA such as: Saturday 11 April 2017 00.37
-    cdata_regex: "\\s*[\\w\\.,:/+-]+(\\s*[\\w\\.,:/+-]+)*\\s*"
+    # Note: Where Unicode characters may exist and a named group such as \w is
+    # used then it's necessary to also include \p{L}\p{N}_.
+    cdata_regex: "\\s*[\\w\\p{L}\\p{N}_\\.,:/+-]+(\\s*[\\w\\p{L}\\p{N}_\\.,:/+-]+)*\\s*"
```

Comment:
JS does not understand this.

```js
const r = /\p{L}/;
r.test('hello'); // => false

Reply:
Yes, which is why we keep `\w` but as you mentioned off channel this now allows for `{` and `}` to be valid which isn't intended though not problematic. We may also want to remove this cdata validation entirely.